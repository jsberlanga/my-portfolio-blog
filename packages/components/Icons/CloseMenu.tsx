import * as React from 'react';

interface CloseMenuProps {
  handleClick: () => void;
}

const CloseMenu: React.FC<CloseMenuProps> = ({ handleClick }) => (
  <svg
    onClick={handleClick}
    version="1.1"
    x="0px"
    y="0px"
    viewBox="2 2 20 20"
    xmlSpace="preserve"
    fill="var(--c-text-02)"
    width="2.4rem"
    height="2.4rem"
    style={{ cursor: 'pointer' }}
  >
    <g>
      <polygon points="18.010437,6.6966553 17.3032837,5.989563 12,11.2928467 6.6967163,5.989563 5.989563,6.6966553    11.2929077,11.999939 5.989563,17.3032837 6.6967163,18.010376 12,12.7070923 17.3032837,18.010376 18.010437,17.3032837    12.7071533,11.999939  " />
    </g>
  </svg>
);

export default CloseMenu;
