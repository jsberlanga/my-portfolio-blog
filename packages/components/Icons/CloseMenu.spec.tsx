import * as React from 'react';
import { render } from '@testing-library/react';
import CloseMenu from './CloseMenu';

describe('CloseMenu', () => {
  it('renders correctly', () => {
    const { container } = render(
      <CloseMenu
        handleClick={() => {
          console.log('click');
        }}
      />,
    );

    expect(container).toMatchSnapshot();
  });
});
