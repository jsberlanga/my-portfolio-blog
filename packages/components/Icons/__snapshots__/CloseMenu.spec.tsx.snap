// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseMenu renders correctly 1`] = `
<div>
  <svg
    fill="var(--c-text-02)"
    height="2.4rem"
    style="cursor: pointer;"
    version="1.1"
    viewBox="2 2 20 20"
    width="2.4rem"
    x="0px"
    xml:space="preserve"
    y="0px"
  >
    <g>
      <polygon
        points="18.010437,6.6966553 17.3032837,5.989563 12,11.2928467 6.6967163,5.989563 5.989563,6.6966553    11.2929077,11.999939 5.989563,17.3032837 6.6967163,18.010376 12,12.7070923 17.3032837,18.010376 18.010437,17.3032837    12.7071533,11.999939  "
      />
    </g>
  </svg>
</div>
`;
